static/js/components/EducationDisplay.js

```diff
@@ -61,18 +61,20 @@ export default class EducationDisplay extends ProfileFormFields {
     );
     return (
       <Cell col={12} className="profile-form-row" key={index}>
-        <div className="basic-info">
+        <div className="basic-info full-width border-bottom">
           <div className="profile-row-name">
-            <span className="school-type">
+            <div className="school-type">
               { degree }
-            </span><br/>
+            </div><br/>
             { entry.school_name }
           </div>
+        </div>
+        <div className="basic-info full-width border-bottom education-top-spacing">
```

Comment:
Basically overall I think this should look more or less like this:

```js
<Cell col={12} className="profile-form-row user-page-row" key={index}>
  <div className="basic-info education">
    <span className="school-type">{ degree }</span>
    <span className="school-name">{ entry.school_name }</span>
  </div>
  <div className="basic-info">
    { userPrivilegeCheck(profile, icons, () => <Cell col={2} />) }
  </div>
</Cell>
```

then for the layout particular to the user page all you need to do is write some CSS targeting `.profile-form-row.user-page-row`. You can do something like this:

```css
.profile-form-row.user-page-row {
  > div {
    width: 50%;
  }
  
  .basic-info.education {
    // education specific styling
  }
  
  .basic-info.work_history {
    // any work-history specific styling
  }
}
```

does that make sense? what you have here is I think just more markup than is necessary.